src/value.rs

```diff
@@ -160,6 +164,14 @@ impl TryFrom<ValueSum> for i64 {
 #[derive(Clone, Debug)]
 pub struct ValueCommitTrapdoor(pallas::Scalar);
 
+impl std::ops::Deref for ValueCommitTrapdoor {
```

Comment:
I'm not sure whether we actually need all these new `Deref` impls (which will be part of the public API); I'd prefer crate-private getters unless we know we will need to unwrap these newtypes outside the crate.

Reply:
I have replaced the `Deref` impls I introduced with crate-private getters.